app/src/common/shared/com/igalia/wolvic/browser/Services.kt

```diff
@@ -92,8 +93,8 @@ class Services(val context: Context, places: Places): WSession.NavigationDelegat
         serverConfig = serverConfig,
         deviceConfig = DeviceConfig(
             // This is a default name, and can be changed once user is logged in.
-            // E.g. accountManager.authenticatedAccount()?.deviceConstellation()?.setDeviceNameAsync("new name")
-            name = "${context.getString(R.string.app_name)} on ${Build.MANUFACTURER} ${Build.MODEL}",
```

Comment:
ah OK so we were already using different names per model, not that bad then :)

Reply:
Yeah, but for Meta Quest 2 it shows as `Oculus Quest` without the 2 here, so this way still has some issues.